.buildkite/pipeline.yml

```diff
@@ -19,6 +19,15 @@ steps:
       automatic:
         limit: 1
       manual: true
+  - name: "Check for docs build :books:"
+    command: ".buildkite/scripts/check_do_docs.sh"
```

Comment:
would it be better to collapse this and the do_docker into 1 step? Will be quicker on average than waiting for more agents/containers to spin up.

Reply:
I'm kinda a fan of being able to do the checks from the high-throughput init agents and not have to wait for a worker to be available. Thoughts?

Reply:
Yeah we’re on the same page. I was suggesting merging this and the previous step. Both are in the init worker so shouldn’t be any slower, and this way you don’t have to wait for init worker slots. Happy to leave as is if you’re not noticing any wait time for this step too.

Reply:
Ahh I misread your q, I thought you meant merge this and the build docs step 🤦‍♂️ . Yeah let me take a look

Reply:
Buildkite supports `commands:` instead of just a single `command:` 👏